### DISCUSSION TITLE
google-cybersecurity-professional-certificate-review/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-programming#50] [Comments]
# google-cybersecurity-professional-certificate-review/

Did you know Google offers a Cybersecurity Professional Certificate? They do! I recently completed it, and wanted to share the good, the bad, and the confusing parts.

https://blog.jakelee.co.uk/google-cybersecurity-professional-certificate-review/

**Comment by ymoudgill8** [comment]
Hello Jake,

Thank you for such an elaborated feedback. I am enrolled in this course and have no experience in the field what so ever, because I would like to learn something new and convert it into a career.

I was wondering if this will help me land an entry level role in the cyber security ? if not what else could you recommend which will compliment the course and also will let me achieve what I want.

Regards
Yogi

**Reply by JakeSteam** [reply]
Hey Yogi, I'm afraid I don't work professionally in cybersecurity so can't provide too much advice. However for all tech jobs, things like this certificate definitely *help*, alongside real world projects, work experience, etc. Good luck!

**Reply by ymoudgill8** [reply]
Thank you for taking time to reply to my comment. I appreciate it.

**Comment by GenxT0** [comment]
Thank you for the detailed write up! I noticed there was a "enroll for free starts on Month, day" on Coursea. Do you need to sign up on specific days? Can you join a class late? Second, does paying monthly for a Coursea subscription include the cost of the google cybersecurity professional exam? Third, can you take the exam whenever you're ready? Lastly, I have experience with infosec but not coding. Will the Python course kill my chances of obtaining this cert? Thanks!

**Reply by JakeSteam** [reply]
Hey! 

1. As far as I know, the signup dates (and deadlines etc) are meaningless, you can just extend / sign up later. I had to extend another course, and it's just 1 click. You're not with a "class", so it doesn't really matter.
2. Yep, Coursera Plus includes all the content / exams, including the final qualifications, that's how I got mine.
3. You can take exams whenever, but I think you also need to pass all the "mini exams" along the way so it'll be quite tricky to just take the required parts.
4. Nope, you can definitely pass this with no prior knowledge! My Python was pretty bad before this, essentially non-existent.

Hope they help!

**Comment by Bcfan121** [comment]
Hey Jake,

Thanks for the review. i am currently doing it and I noticed how easy it was for me to skip the courses but the layout seems to very similar to Linkedin Learning. i wish they would get a human to check it for you then it would be much more challenging to pass but I did notice that they did change google certificate now you cannot move to the next steps if you finish the labs but I did notice that you could open the Python labs and save it and then the system would allow the user to move to the next stage. I might report this issue with them. Personally, I think that the Certificate IV Cybersecurity I am doing seems a bit more better and harder to finish because it's being marked by a human too I noticed how easily someone could just copy & paste these multiple choice answers onto Chatgpt and let chatgpt answer it for them. I only have 2 more to go which one do you recommend I should do next? thanks

**Reply by JakeSteam** [reply]
Up to you! I did the Google UX Design course next as I was interested in it, I'm not a cybersecurity expert unfortunately.

**Reply by Bcfan121** [reply]
I have not received any improvement after undertaking this certificate and TSM as well.

**Comment by ckk0011** [comment]
I have done this coursera Google cybersecurity program through Merit America....Merit America promotes these as career changing certificates...after close to 300 applications with around 50% rejection, and 50% no response I don't think it is worth the time or the cost, It could be a decent prep for a CompTIA certificate, but don't think those are worth the time alone, without a degree to back it up.

**Reply by Bcfan121** [reply]
I have not received any improvement after undertaking this certificate and TSM as well. I actually asked Coursera for my money back as well and degree costs almost as expensive as a house these days.

**Reply by ckk0011** [reply]
Your telling me I’ll have a ton of money into a degree when finished

**Comment by thAbstract** [comment]
Thank you for the VERY THOROUGH review!  I have read a Google cert wont really land you a job, BUT at least the material in the Cybersecurity Professional Certification course is relevant and should give newbies a better knowledge base, if nothing else.

**Comment by JayasooryaPS** [comment]
Hey Lee!! Thankyou for posting your blog. Is the certificate you uploaded the updated one. Because my Google Data Analysis certificate in coursera was different and didn't have bried information about the courses.

**Reply by JayasooryaPS** [reply]
*brief information

**Comment by 3L4CKBURN** [comment]
The DEI part was spot on,also that lady with lip piercing was disgusting for me( only the piercing)
Also saying anyone can do it was kinda lame .
